public/package.json

```diff
@@ -5,8 +5,8 @@
   "homepage": "",
   "author": "Bitovi",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1",
-    "install": "node build.js"
+    "install": "node build.js",
+    "test": "testee test.html --browsers firefox --reporter Spec"
```

Comment:
Use Firefox. Phantom sucks. 

Sent from my iPhone

> On Feb 15, 2016, at 2:22 PM, Chris Gomez <notifications@github.com> wrote:
> 
> In public/package.json:
> 
> > @@ -5,8 +5,8 @@
> >    "homepage": "",
> >    "author": "Bitovi",
> >    "scripts": {
> > -    "test": "echo \"Error: no test specified\" && exit 1",
> > -    "install": "node build.js"
> > +    "install": "node build.js",
> > +    "test": "testee test.html --browsers firefox --reporter Spec"
> I would have assumed we'd use Phantom since we can add it as an NPM dependency. Firefox may not be installed by the developer.
> 
> —
> Reply to this email directly or view it on GitHub.
>